docs/high-availability/kube-apiserver.manifest

```diff
@@ -0,0 +1,103 @@
+  {
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-apiserver"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-apiserver",
+    "image": "jayunit100/ha-kube-api:1",
```

Comment:
this should be a release version preferably hosted on grc.io. what does version 1 mean? why is it named differently?